### PR TITLE
Extract more strings for localization

### PR DESCRIPTION
**Pull request by GuilleHoardings**
Only English and Spanish translations added.

### CODE REVIEW

File: website/src/components/Buttons/Skip.tsx [hunk 1]
```diff
@@ -11,13 +11,15 @@ import {
   Textarea,
   useDisclosure,
 } from "@chakra-ui/react";
+import { useTranslation } from "react-i18next";
```
Comment (AbdBarho) [inline website/src/components/Buttons/Skip.tsx:14]: `"next-i18next"`?

**Review by AbdBarho — COMMENTED** [review]
Thanks!

File: website/src/components/Buttons/Skip.tsx (hunk 1 above)
Comment (GuilleHoardings) [reply website/src/components/Buttons/Skip.tsx]: Oops, yes. Fixed!

File: website/src/components/Survey/LabelLikertGroup.tsx [hunk 2]
```diff
@@ -18,160 +19,154 @@ interface LabelInfo {
   inverted: boolean;
 }
 
-const getLabelInfo = (label: string): LabelInfo => {
+const getLabelInfo = (label: string, t: (key: string) => string): LabelInfo => {
```
Comment (notmd) [inline website/src/components/Survey/LabelLikertGroup.tsx:22]: This is not typesafe? Could you look more into it? Mabe the following will work
`t: ReturnType<typeof useTranslation>["t"]`

Comment (GuilleHoardings) [reply website/src/components/Survey/LabelLikertGroup.tsx]: I've changed it. I don't know typescript, so I don't know if it's right. It seems to work.

Comment (GuilleHoardings) [reply website/src/components/Survey/LabelLikertGroup.tsx]: It looks like the E2E tests have failed. I've taken a look, but I don't understand why, sorry.

File: website/src/components/Buttons/Skip.tsx [hunk 3]
```diff
@@ -11,13 +11,15 @@ import {
...
 } from "@chakra-ui/react";
+import { useTranslation } from "next-i18next";
 import { useState } from "react";
 
 interface SkipButtonProps extends ButtonProps {
   onSkip: (reason: string) => void;
 }
 
 export const SkipButton = ({ onSkip, ...props }: SkipButtonProps) => {
+  const { t } = useTranslation("common");
```
Comment (rjmacarthy) [inline website/src/components/Buttons/Skip.tsx:22]: I'm not sure, but maybe you don't need to specify the namespace when the only one used is `common`?

File: website/src/components/Survey/TaskControls.tsx [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import { Box, Flex, IconButton, Progress, Tooltip, useColorModeValue } from "@chakra-ui/react";
+import { useTranslation } from "react-i18next";
```
Comment (rjmacarthy) [inline website/src/components/Survey/TaskControls.tsx:2]:
```suggestion
import { useTranslation } from "next-i18next";
```

File: website/src/components/TaskInfo/TaskInfo.tsx [hunk 5]
```diff
@@ -1,4 +1,5 @@
 import { Box, Flex, Text, TextProps, useColorModeValue } from "@chakra-ui/react";
+import { useTranslation } from "react-i18next";
```
Comment (rjmacarthy) [inline website/src/components/TaskInfo/TaskInfo.tsx:2]:
```suggestion
import { useTranslation } from "next-i18next";
```

File: website/public/locales/en/common.json [hunk 6]
```diff
@@ -2,6 +2,7 @@
   "about": "About",
   "account_settings": "Account",
   "admin_dashboard": "Admin Dashboard",
+  "any_feedback_on_this_task": "Any feedback on this task?",
```
Comment (rjmacarthy) [inline website/public/locales/en/common.json:5]: Seems like not a common utterance, maybe better off in `tasks.json`?

File: website/src/components/Buttons/Skip.tsx (hunk 3 above)
Comment (GuilleHoardings) [reply website/src/components/Buttons/Skip.tsx]: You are right. I've changed it in all the files that only used `common`.

File: website/src/components/Survey/TaskControls.tsx (hunk 4 above)
Comment (GuilleHoardings) [reply website/src/components/Survey/TaskControls.tsx]: Thank you. I've fixed all the `"react-i18next" ` that I added in my commit. However, there's still one in `src/pages/account/index.tsx` that was inserted by @AbdBarho and I don't know if it should be changed.

File: website/src/components/TaskInfo/TaskInfo.tsx (hunk 5 above)
Comment (GuilleHoardings) [reply website/src/components/TaskInfo/TaskInfo.tsx]: See previous comment.

File: website/src/components/Survey/TaskControls.tsx (hunk 4 above)
Comment (notmd) [reply website/src/components/Survey/TaskControls.tsx]: > Thank you. I've fixed all the `"react-i18next" ` that I added in my commit. However, there's still one in `src/pages/account/index.tsx` that was inserted by @AbdBarho and I don't know if it should be changed.

Yes it should be update too

File: website/public/locales/en/common.json (hunk 6 above)
Comment (GuilleHoardings) [reply website/public/locales/en/common.json]: Makes sense. I've updated it. Thank you again!

File: website/src/components/Survey/TaskControls.tsx (hunk 4 above)
Comment (GuilleHoardings) [reply website/src/components/Survey/TaskControls.tsx]: Changed. Thank you!

File: website/src/components/Survey/LabelLikertGroup.tsx (hunk 2 above)
Comment (GuilleHoardings) [reply website/src/components/Survey/LabelLikertGroup.tsx:22]: @notmd I've been working on this. I've found that the solution you proposed causes an infinite loop in the type checking and that was the cause of the E2E test failing.

I've gone back to my initial solution. I think that it is type safe. As I understand it, it's saying that parameter `t` is a function that receives a `string` and returns a `string`. Isn't that correct?

Comment (notmd) [reply website/src/components/Survey/LabelLikertGroup.tsx:22]: So I'm fine with the current behavior. Thanks

**Review by fozziethebeat — APPROVED** [review]

**Comment by fozziethebeat** [comment]
It looks like all change requests have been applied or at least attempted.  Merging in.

**Pull request merged**